pkg/hooks/read_test.go

```diff
@@ -171,27 +171,6 @@ func TestUnknownDir(t *testing.T) {
 	}
 }
 
-func TestBadDir(t *testing.T) {
```

Comment:
Why are we dropping this? I think we should still fail on hooks with an invalid version

Reply:
In order to handle Ed's vision which is more future proof, we had to drop this test.  Let's discuss today. BTW Maybe we should just merge the original and then talk about this at standup.

Reply:
Maybe my suggestion _isn't_ more futureproof. Maybe if there's a v2.0.0 or v1.2.3 it will be incompatible enough that podman _should_ abort early. I have no sense for these hooks, for the json format, for when/how that format changes. I just think we need to think about it in advance.

Reply:
If we had a more easily parsed version number, I'd say we should accept anything with major version 1 and deny anything else, assuming semantic versioning.

Reply:
Is there a (relatively easy) workaround that can implemented in the meantime, for users to be able to launch containers on their systems in the meantime?

Reply:
@heatmiser you already tried adding `"version":"0.1.0",` to the json... I wonder, could you try `"1.0.0"`? I apologize for not trying it myself first.

Reply:
Version | Error
-- | --
"0.1.0" | error setting up OCI Hooks ... unrecognized hook version: "0.1.0"
"0.1.1" | error setting up OCI Hooks ... unrecognized hook version: "0.1.1"
"1.0.0" | error setting up OCI Hooks ... 1.0.0: json: cannot unmarshal string into Go struct field Hook.hook of type specs.Hook

Fedora 31
`# podman -v`
`podman version 2.0.0-dev`
`# rpm -q podman`
`podman-2.0.0-1592321502.git908bc3fb.fc31.x86_64`
...from Copr repo for Upstream_CRIO_Family owned by baude
Recently executed package updates, not sure what previous version of podman was, but it worked
Note that I am also seeing the same error messages for `/usr/share/containers/oci/hooks.d/oci-systemd-hook.json`  as I am for `/usr/share/containers/oci/hooks.d/oci-umount.json` when I attempt to start containers via podman.